src/python/benchUtil.py

```diff
@@ -1381,17 +1399,26 @@ def simpleReport(self, baseLogFiles, cmpLogFiles, jira=False, html=False, baseDe
       else:
         w('%16s' % ('%7.1f%% (%4d%% - %4d%%)' % (psAvg, psWorst, psBest)))
 
+      if jira:
+        w('|%s' % (jiraColor(pValue)))
+      elif html:
+        w('<td>%s</td>' % htmlColor2(pValue))
```

Comment:
Oh yeah!   You are already using colors for HTML mode!  Thanks.  Hmm, but what is this `htmlColor2` doing?

Reply:
I have no idea! I just copy paste cargo-cult programming style... I see now, looking at these color methods that they do not compare against the right threshold for p-value so they will probably all be red or green depending on which method is used> I'll follow up with a color patch although I don't know how this is used exactly -- ah I see there is a -jira argument and -html ...